packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -318,6 +338,18 @@ export function GraphiQLInterface(props: GraphiQLInterfaceProps) {
     isChildComponentType(child, GraphiQL.Footer),
   );
 
+  const tabName = (currentTab: String) => {
+    if (currentTab === 'variables') {
+      return 'Variables';
+    }
+    if (currentTab === 'extensions') {
+      return 'Extensions';
+    }
+    if (currentTab === 'headers') {
+      return 'Headers';
+    }
+  };
```

Comment:
This looks like a pretty basic method, I thought I'd make it as simple as possible and not be clever with a .capitalize

Previously this was a ternary in the component, which was fine when there were only 2 tabs.

Reply:
nice that works. and it's still in the component scope so we can readily translate it later